aten/src/THC/THCTensorMathPointwise.cuh

```diff
@@ -462,22 +476,22 @@ struct TensorCRemainderOp {
 template <>
 struct TensorCRemainderOp<float> {
   __device__ __forceinline__ void operator()(float* out, float* in) {
-    *out = *in != 0 ? *out - *in * floorf(*out / *in) : NAN;
+    *out = *in != 0.f ? *out - *in * floorf(*out / *in) : NAN;
```

Comment:
shouldn't 0 be okay in these cases?  It should automatically be cast to the right type?

Reply:
It probably is. I only changed it to be more consistent, it's not part of the fix really.